esi_leap/common/statuses.py

```diff
@@ -20,4 +20,7 @@
 WAIT_EXPIRE = 'wait expire'
 WAIT_FULFILL = 'wait fulfill'
 
+OFFER_CAN_DELETE = ('available', 'error')
```

Comment:
You'll want to use AVAILABLE and ERROR here instead of duplicating strings. Any reason we're using '(' instead of '['?

Reply:
Yeah, sam_z suggested that since there are only two values, it should be a tuple rather than list so I changed '[' to '('.
Should I changed it back?

Reply:
I don't think it's a big deal either way; you can just leave it!